src/services/ledger/services/transactions-metadata.ts

```diff
@@ -1,6 +1,6 @@
-import { NoTransactionToUpdateError, UnknownRepositoryError } from "@domain/errors"
+import { NoTransactionToUpdateError } from "@domain/errors"
 import { CouldNotFindTransactionMetadataError } from "@domain/ledger"
-import { fromObjectId, toObjectId } from "@services/mongoose/utils"
+import { fromObjectId, toObjectId, parseRepositoryError } from "@services/mongoose/utils"
```

Comment:
This isn't great but as this is also a repository implemented with `mongoose` its okay as an exception.